public/app/features/logs/components/LogRowContext.tsx

```diff
@@ -48,7 +48,7 @@ const getLogRowContextStyles = (theme: GrafanaTheme2, wrapLogMessage?: boolean)
         top: 100%;
       `
     : css`
-        margin-top: 40px;
+        margin-top: 20px;
```

Comment:
What about using `theme.spacing()` or similar?

Reply:
Good suggestion, I think there's a dedicated issue to tackle this: https://github.com/grafana/grafana/issues/55949